tests/sanic/app.py

```diff
@@ -0,0 +1,32 @@
+from random import random
+
+import strawberry
+from sanic import Sanic
+from strawberry.file_uploads import Upload
+from strawberry.sanic.views import GraphQLView as BaseGraphQLView
+
+
+def create_app(**kwargs):
+    @strawberry.type
+    class Query:
+        hello: str = "strawberry"
+
+    @strawberry.type
+    class Mutation:
+        @strawberry.mutation
+        def read_text(self, text_file: Upload) -> str:
+            return text_file.read().decode()
+
+    schema = strawberry.Schema(query=Query, mutation=Mutation)
+
+    class GraphQLView(BaseGraphQLView):
+        def get_root_value(self):
+            return Query()
+
+    app = Sanic("test-app-" + str(random()))
+
+    app.add_route(
+        GraphQLView.as_view(schema=schema, graphiql=kwargs.get("graphiql", True)),
```

Comment:
When will `kwargs` be populated? Tests don't cover it

Reply:
One of te existing Sanic tests makes use of it:

https://github.com/strawberry-graphql/strawberry/blob/f31e6c17238ba4372129c4f291ddf336b4e6b901/tests/sanic/test_view.py#L58-L62